manifests/m/Microsoft/VC++2015-2019Redist-x64/14.29.30037.0/Microsoft.VC++2015-2019Redist-x64.installer.yaml

```
@@ -0,0 +1,24 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Microsoft.VC++2015-2019Redist-x64
+PackageVersion: 14.29.30037.0
+MinimumOSVersion: 10.0.0.0
+InstallerSuccessCodes:
+  - 3010
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+Installers:
+  - Architecture: x64
+    InstallerType: exe
+    InstallerUrl: https://aka.ms/vs/16/release/vc_redist.x64.exe
```

```suggestion
    InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/f1998402-3cc0-466f-bd67-d9fb6cd2379b/A1592D3DA2B27230C087A3B069409C1E82C2664B0D4C3B511701624702B2E2A3/VC_redist.x64.exe
```
This shooould be the Static URL